context.go

```diff
@@ -74,7 +88,12 @@ func (context *Context) Run(addressesToData map[string]any, timeout time.Duratio
 		containerMaxSize: wafMaxContainerSize,
 		objectMaxDepth:   wafMaxContainerDepth,
 	}
-	obj, err := encoder.Encode(addressesToData)
+	persistentData, err := encoder.Encode(addressData.persistent)
+	if err != nil {
+		return res, err
+	}
+
+	ephemeralData, err := encoder.Encode(addressData.ephemeral)
```

Comment:
You will have to find a way to bypass the usual behaviour of `Encode` because in case the input is nil it will return a `errUnsupportedValue`.

Reply:
Same as https://github.com/DataDog/go-libddwaf/pull/39#discussion_r1381969290 -- I think that is not an issue?